constants.js

```diff
@@ -1,35 +0,0 @@
-const { mapValues, orderBy } = require('lodash');
```

Comment:
Why this file is removed? CLIENT_SECRET is still referenced in .dev, .stage & .production env-files?

Reply:
Removed from env files